setup/docker.md

```diff
@@ -158,6 +158,40 @@ multi-machine scaling.
 
 For organizations, we recommend one Docker host per team of 5-10 developers.
 
+## Docker Compose
+
+   For a convenent way to run Coder in Docker, you can use [Docker Compose](https://docs.docker.com/compose/).
+
+   Simply create a new directory (for example, named `c4d`) and place the following
+   content in a file named `docker-compose.yml`:
+
+   ```yaml
```

Comment:
Do we need the extra spaces here?
```suggestion
```yaml
```

Reply:
The linter says yes, so I did what it said.

Reply:
I just tried keeping the current indentation level; is there anything magical about that here?

Reply:
Ah, maybe my eyes deceive me - I thought this line was off compared to the rest. Disregard!